ocp_resources/rhmi.py

```diff
@@ -8,3 +9,43 @@ class RHMI(NamespacedResource):
     """
 
     api_group = NamespacedResource.ApiGroup.INTEGREATLY_ORG
+
+    class Status:
+        INSTALLATION = "installation"
+        COMPLETE = "complete"
+
+    def wait_for_status_complete(self, timeout):
```

Comment:
why not use  `wait_for_status`?

Reply:
From what I understand `wait_for_status()` waits for `status.phase`, but in RHMI we need to check `status.stage` value, and for that I added this function. 
Can we use the same function in this case also?

Reply:
I guess not.